### DISCUSSION TITLE
Unable to Sign in into Github because of MFA

### DISCUSSION THREAD

**Discussion by sanyam1997** [community/community#40854] [Other Feature Feedback, Questions, & Ideas]
### Explanation

I had MFA setup for github via DUO and I lost the recovery codes and my phone was broken so I can't use my duo MFA either.
I don't have any other recovery email so I can't use that as a recovery method either.
Can anyone please help?
Sanyam

**Comment by airtower-luna** [comment]
See [Recover an account with 2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials) for other options you have. I'm afraid if none of those work you can't get the account back, a staff member stated that [here](https://github.com/orgs/community/discussions/39629#discussioncomment-4196863), and the documentation warns about it, too.

**Reply by sanyam1997** [reply]
Yes, it seems like my account isn't recoverable. Do you think its a lost cause?
Can I delete my account somehow so I can use the same email to create a new github account?

**Comment by mohamedzhioua** [comment]
check this two links and i wish you reach your needs :"https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials"
 "https://guide.duo.com/duo-restore"
